### DISCUSSION TITLE
Robots.txt unreachable - Google bots couldn't get the sitemap from the vercel

### DISCUSSION THREAD

**Discussion by raja-vignesh** [vercel/community#2803] [Bug Report]
### Page to Investigate

https://quizzesforyou.com/robots.txt

### Steps to Reproduce

Live URL Test in Google Search Console fails due to Robots.txt being unreachable in the Page.

It was working till yesterday. It's related to
https://support.google.com/webmasters/thread/216797133/availability-fails-due-to-robots-txt-unreachable?msgid=220044677

**Comment by smaeda-ks** [comment]
@raja-vignesh can you join https://github.com/orgs/vercel/discussions/2611#discussioncomment-6047511 ?

I think it started happening again. We're reaching out to Google again
